BridgeAppSDK/SBAProfileDataSource.swift

```diff
@@ -71,3 +74,66 @@ public protocol SBAProfileDataSource: class {
     @objc(titleForSection:)
     optional func title(for section: Int) -> String?   
 }
+
+open class SBAProfileDataSourceObject: SBADataObject, SBAProfileDataSource {
+    
+    /**
+     The shared instance of the profile data source. Loads from SBAProfileJSONFilename, which
+     defaults to "Profile".
+     
+     If you subclass SBAProfileDataSourceObject, or just want this method to return a different class that
+     implements the SBAProfileDataSource protocol, you will need to override the mapping for the default
+     value of SBAProfileDataSourceClassType ("ProfileDataSource") in your ClassTypeMap.plist to map to your
+     class instead of this one. Alternatively you could change the value of SBAProfileSectionClassType
+     before accessing this, and map *that* in your ClassTypeMap.plist; or just set SBAProfileSectionClassType
+     to the fully qualified class name without setting up any mappings.
+     */
+    public static let shared: SBAProfileDataSource? = {
```

Comment:
It's confusing to declare this as optional and then force it when you use the lazy initializer below without any explanation for why that is throwing an error.  Instead of returning nil can you do this:

````
else {
    assertionFailure("Couldn't find the shared profile data source.")
    return SBAProfileDataSourceObject(dictionary: [:])
}
````

Or if it makes more sense, just return an empty source without an assert and then have the lazy initializer throw an assert if the number of sections == 0.